app/controllers/application_controller.rb

```diff
@@ -1,5 +1,12 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
+  protect_from_forgery with: :exception
+
+  before_action :update_allowed_parameters, if: :devise_controller?
+
```

Comment:
- [ ] To make the devise gem work with your app, Please add `before_action :authenticate_user!` here in your `application_controller` because now nothing appears when you run rails server.